gradle.properties

```diff
@@ -17,7 +17,7 @@ digraphVersion = 1.0
 disklrucacheVersion = 2.0.2
 hamcrestCoreVersion = 1.3
 jacksonVersion = 2.9.8
-jcommanderVersion = 1.74
+jcommanderVersion = 1.75
```

Comment:
I get `java.lang.NoClassDefFoundError: com/beust/jcommander/JCommander` when I try this branch locally, I saw the same error on CI. The JAR of 1.75 contains basically the same files as 1.74, except for an empty `kobaltBuild` directory and some OSGI info in the manifest. The class files have the same sizes, but are not identical.

Reply:
The manifest of 1.75 is also broken:
https://github.com/cbeust/jcommander/commit/85cb6f7217e15f62225185ffd97491f34b7b49bb#commitcomment-32810824

Reply:
I saw that comment, but interestingly a local build succeeded for me, so I wanted to wait until I also see it fail on CI myself.

Reply:
[Confirmed](https://travis-ci.com/heremaps/oss-review-toolkit/builds/106141348#L733), I've dropped the JCommander upgrade commit.